packages/aws-cdk/package.json

```
@@ -14,6 +14,8 @@
     "pkglint": "pkglint -f",
     "test": "cdk-test",
     "integ": "jest --testMatch '**/?(*.)+(integ-test).js'",
+    "bundle": "esbuild bin/cdk.ts --bundle --platform=node --external:fsevents --outfile=bin/cdk-bundle.js --allow-overwrite && cp ../../node_modules/vm2/lib/contextify.js bin",
```

You'll want to also specify `--target=node12` to ensure compatibility with our supported runtimes (the default is otherwise `esnext`, which might allow syntax that some supported runtimes don't support)

```suggestion
    "bundle": "esbuild bin/cdk.ts --bundle --platform=node --target=node12 --external:fsevents --outfile=bin/cdk-bundle.js --allow-overwrite && cp ../../node_modules/vm2/lib/contextify.js bin",
```

---

What/why is this:

```
cp ../../node_modules/vm2/lib/contextify.js bin
```

---

Done. 

> What/why is this:
> cp ../../node_modules/vm2/lib/contextify.js bin

Because its a dynamically loaded file by our transitive dependency `vm2`. (was renamed to `bridge.js` now).